src/daemon.cpp

```diff
@@ -29,7 +29,7 @@ void close_server() {
 void do_heartbeat()
 {
     // accept multiple connections - I noticed that new clients are only accepted when the primary client writes to the server
-    if(server->accept() != -1) // accepts any incoming connection
+    if(server->accept()) // accepts any incoming connection
```

Comment:
This returns a bool, so it made no sense to compare with -1. I think this is what you meant. Correct?

Reply:
Right. That was a mistake on my part.